nym-vpn-core/Makefile

```diff
@@ -24,9 +27,10 @@ build-mac: ## Build the Rust workspace suitable for running the daemon
 build-win-cross:
 	cargo build --target x86_64-pc-windows-gnu --release
 
-deb: build-deb-vpn-cli build-deb-vpnd build-deb-vpnc ## Build debian packages
+# -----------------------------------------------------------------------------
+#  Linting targets
+# -----------------------------------------------------------------------------
 
-# Linting targets
```

Comment:
I think we had a script parsing these as a part of help for make. Is it still functional?

Reply:
Ah you're right, accidentally remove the one for `deb:`. Restored now